tests/plugins/dialog/manual/opennonfirsttabasdefault.html

```diff
@@ -0,0 +1,49 @@
+<textarea name="" id="editor" cols="30" rows="10">
+	hello world
+</textarea>
+<button id="press">Open Dialog</button>
+<script>
+	var editor = CKEDITOR.replace( 'editor' );
+
+	CKEDITOR.once( 'instanceLoaded', function() {
+		CKEDITOR.dialog.add( 'testDialog', function() {
+			return {
+				title: 'Test dialog',
+				contents: [
+					{
+						id: 'tab1',
+						label: 'Tab one',
+						elements: [
+							{
+								type: 'html',
+								id: 'field11',
+								html: 'foo'
+							}
+						]
+					},
+					{
+						id: 'tab2',
+						label: 'Tab two',
+						elements: [
+							{
+								type: 'html',
+								id: 'field21',
+								html: 'bar'
+							}
+						]
+					}
+				]
+			}
+		} );
+
+		editor.addCommand( 'testDialog', new CKEDITOR.dialogCommand( 'testDialog', {
+			tabId: 'tab2'
+		} ) );
+
+	} );
+
+	editor.on( 'dialogShow', function() { console.log( 'Show' ) } );
```

Comment:
Is this `console.log` needed here? If yes, please include checking console output as a scenario step, if not simple remove it.

Reply:
I removed it.
It left after my checking if `dialogShow` event is execute inside openDialog method.